### PR TITLE
Add app description to app popup

### PR DESCRIPTION
**Pull request by yathindrakodithuwakku**
### Related Issues
- 

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on final implementation.
- [ ] Documentation provided. (Add links if there's any)
- [ ] Unit tests provided. (Add links if there's any)
- [ ] Integration tests provided. (Add links if there's any)

### Related PRs
- Related PR `#1` or (None)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

**Comment by yathindrakodithuwakku** [comment]
The updated UI is shown here

Screenshot 2022-01-10 at 14 15 09

File: apps/myaccount/src/components/applications/application-list.tsx [hunk 1]
```diff
@@ -131,7 +131,14 @@ export const ApplicationList: FunctionComponent<ApplicationListProps> = (
 
                                         ) }
                                         position="top center"
-                                        content={ app.name }
+                                        content={ (
+                                            <div className="row">
```
Comment (brionmario) [inline apps/myaccount/src/components/applications/application-list.tsx:135]: Can't we use `Grid` component here?
```suggestion
                                            <Grid.Row>
```

File: modules/theme/src/themes/default/views/item.overrides [hunk 2]
```diff
@@ -90,3 +90,8 @@
         }
     }
 }
+
+.hint-description {
+    color: @applicationListItemDescColor;
+    margin-top: "0.2rem"
+}
```
Comment (brionmario) [inline modules/theme/src/themes/default/views/item.overrides:97]:
```suggestion
}

```

File: apps/myaccount/src/components/applications/application-list.tsx [hunk 3]
```diff
@@ -131,7 +131,14 @@ export const ApplicationList: FunctionComponent<ApplicationListProps> = (
...
+                                                <div className="column">{ app.name?.length > 55 ? 
+                                                app.name?.substring(0, 56) + " ..." : app.name }</div>
+                                                <div className="column hint-description">{ app.description?.length > 55 ?
```
Comment (brionmario) [inline apps/myaccount/src/components/applications/application-list.tsx:138]: Check if you can use `Text` from `react-components` here.
```jsx
<Text muted>YOUR TEXT</Text>
```

**Review by brionmario — APPROVED** [review]

**Review by yasinmiran — APPROVED** [review]

**Pull request merged**